Packet++/src/BgpLayer.cpp

```diff
@@ -744,9 +745,25 @@ namespace pcpp
 
 		if (newNlriDataLen > curNlriDataLen)
 		{
+
+			// offsetInLayer, numOfBytesToExtend
+			//		int indexToInsertData = layer->m_Data + offsetInLayer - m_RawPacket->getRawData();
+			auto bytesToExtend = newNlriDataLen - curNlriDataLen;
+
+			if (m_Data != nullptr && m_Packet != nullptr)
+			{
+				auto raw_len = static_cast<size_t>(m_Packet->getRawPacket()->getRawDataLen());
```

Comment:
Variable name convention is camleCase: `rawLen` instead of `raw_len`

Reply:
ah, had the wrong impression due to the `m_`, my bad